metacontroller/base/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: meta-controller-cluster-role-binding
```

Comment:
you could probably use namePrefix: meta-controller- in the kustomization.yaml and then this would just be cluster-role-binding